src/soap/handlers/GetSites.js

```diff
@@ -48,12 +48,16 @@ export async function* getSites(
       : site && typeof site === 'object' && Array.isArray(site.string)
       ? site.string.filter(str => !!str)
       : []
+  const org =
+    typeof request.params.org === 'string'
+      ? helpers.org(request.params.org)
+      : undefined
 
   // Fetch organization
   const organization =
-    typeof request.params.org === 'string'
+    typeof org === 'string'
```

Comment:
Could probably just to a truthy test since the string check is above. Maybe the general rule for all of these handlers is type checks and params conversion first, then just do simple `org ? await ... : undefined` style check everywhere below. At this point we know that org is either a string or undefined. No need to check for string again.